README.md

```diff
@@ -1,2 +1,51 @@
-# comp-mode-usb
-An implementation of the Metadata USBs in CircuitPython using an Adafruit Neo Trinkey
+# Competition Mode USB
+An implementation of the [Metadata USBs](https://srobo.github.io/astoria/usage.html#metadata-usbs) in CircuitPython using an Adafruit Neo Trinkey.
+
+# Configuring a Neo Trinkey
+
+The Neo Trinkey does not come with the CircuitPython bootloader installed.
+
+To install the bootloader, plug the Trinkey and double press the reset button.
+All 4 LEDs should now be green and a TRINKEYBOOT drive should be visible, if they are red there is a problem with the USB connection.
+Download the CircuitPython bootloader from the [circuitpython website](https://circuitpython.org/board/neopixel_trinkey_m0/) and drag the uf2 file onto the TRINKEYBOOT drive.
+The Neo Trinkey will then reboot and a CiRCUITPY drive will appear.
```

Comment:
Question: "CiRCUITPY" or "CIRCUITPY"? (capital "I")

Reply:
My pinky got tired. Yes, it's all caps.

Reply:
bef71f6b627e579cfcf261b9e205bf0b7bf3cf97